js/webpack_get_defaults.js

```diff
@@ -132,7 +139,7 @@ module.exports = function (basePath) {
       loaders: [
         {
           test: /\.js/,
-          loader: combineLoaders(jsLoaders),
+          loader: 'babel?{"presets":["es2015","stage-3"],"plugins":["transform-strict-mode",["transform-runtime",{"polyfill":true,"regenerator":false}]]}',
```

Comment:
I've had an issue with `combineLoaders`. The output url was not recognized by babel loader, so I ended up using the JSON format for tests purposes. Do you guys have any idea why the stringification by `combineLoaders` has failed ?